phylorthology/tsar_downsamp_test_parameters.csv

```diff
@@ -0,0 +1,12 @@
+{
+  "input": "https://github.com/Arcadia-Science/tsar-phylorthology/blob/main/samplesheets/downsamp_test_tsar_samplesheet.csv",
+  "outdir": "results-downsamp-tsar-test",
+  "mcl_inflation": "1.0,2.0,3.0,4.0,5.0",
+  "min_num_spp_per_og": "4",
```

Comment:
These should be numbers not strings I think? Either the pipeline defaults are wrong or these are wrong.

Reply:
Nope, you're right

Reply:
Fixed that, and also simplified the tree inference (i.e. no pmsf tree model)